### PR TITLE
Improvement/s3 c 3971 reduce event footprint

### PR DESCRIPTION
**Pull request by tmacro**
Includes an update to the latest arsenal to make the cloudserver update less painful

### CODE REVIEW

File: libV2/tasks/IngestShard.js [hunk 1]
```diff
@@ -47,16 +56,16 @@ class IngestShardTask extends BaseTask {
                         if (shardAge < checkpointLagMicroseconds) {
                             metricClass = 'utapi.event';
                             records = metrics
-                                .map(m => new UtapiMetric(JSON.parse(m)));
+                                .map(m => this._hydrateEvent(m));
```
Comment (miniscruff) [inline libV2/tasks/IngestShard.js:59]: is it possible to remove the extra function wrap?
```suggestion
                                .map(this._hydrateEvent);
```

File: tests/functional/v2/task/testIngestShards.js [hunk 2]
```diff
@@ -123,5 +126,23 @@ describe('Test IngestShards', function () {
         );
         assertResults(events, series);
     });
+
+    it('should strip the event uuid during ingestion', async () => {
+        const start = shardFromTimestamp(getTs(-120));
+        const stop = start + 9000000;
+        const events = generateFakeEvents(start, stop, 100);
+
+        await Promise.all(events.map(e => cacheClient.pushMetric(e)));
```
Comment (miniscruff) [inline tests/functional/v2/task/testIngestShards.js:135]: might also be able to remove the function wrap here too
```suggestion
        await Promise.all(events.map(cacheClient.pushMetric));
```

**Review by miniscruff — APPROVED** [review]

File: libV2/tasks/IngestShard.js (hunk 1 above)
Comment (tmacro) [reply libV2/tasks/IngestShard.js:59]: You can do something like `this._hydrateEvent.bind(this)`, but it's just doing the same thing with a different syntax. Root problem is that js doesn't carry around a reference to the correct `this`, so in `_hydrateEvent` on L#27 without the wrap/bind `this` would not be pointing to an instance of `IngestShardTask`.

File: tests/functional/v2/task/testIngestShards.js (hunk 2 above)
Comment (tmacro) [reply tests/functional/v2/task/testIngestShards.js:135]: See above

File: libV2/tasks/IngestShard.js (hunk 1 above)
Comment (miniscruff) [reply libV2/tasks/IngestShard.js:59]: right... that dang `this`...

**Review by ilkescality — APPROVED** [review]

**Pull request merged**